workspaces/todo/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "todo",
   "version": "1.0.0",
```

Comment:
Are we commiting to everything being 1.0.0? or should it be following the same version numbers?

Reply:
should be same version number as the `@backstage` one.